lib/models/conversation_resource_response.ex

```diff
@@ -0,0 +1,34 @@
+defmodule ExMicrosoftBot.Models.ConversationResourceResponse do
+  @moduledoc """
+  Microsoft bot conversation API resource response
+  """
+
+  @derive [Poison.Encoder]
+  defstruct [:id, :activityId]
```

Comment:
```suggestion
  defstruct [:id, :activityId, :serviceUrl]
```

Reply:
What if this field is absent in response? Will it cause exceptions?

Reply:
No, it should be nil.